client/state/index.js

```diff
@@ -39,10 +38,23 @@ const reducer = combineReducers( {
 	ui
 } );
 
-var createStoreWithMiddleware = applyMiddleware(
-	thunkMiddleware,
-	analyticsMiddleware
-);
+let middleware = [ thunkMiddleware ];
+
+if ( typeof window === 'object' ) {
```

Comment:
Maybe add a comment above that this is for SSR which that middleware's `analytics` dependecy would break?
Aside: Do we want some global `isServer()` helper thingy instead of those `window` checks all over? (Not sure where that would live...)